django/core/templates/snippets/error-content.html

```diff
@@ -0,0 +1,21 @@
+<div class="errorpage-content">
+    
+    {# Requires the path to be known to confirm it to the user #}
+    {% if request.path %}
+        <p>
+            We couldn't find anything at the location <span class="errorpage-path">{{ request.path }}</span>
+        </p>
+    {% endif %}
+    
+    <p>
+        To return to a working page, please click back in your browser or go to the home page.
+    </p>
+
+    {# Requires a path to be known in order to email admin #}
+    {% if request.path %}
+        <p>
+            <a href="mailto:k.mcauliffe@bham.ac.uk?subject=EUCLCORP website (Error)&body=I'd like to report an error at the location: ' {{ request.path }} ' on the EUCLCORP website">k.mcauliffe@bham.ac.uk</a>
```

Comment:
Is there a way to specify the email in local_settings.py - then it wouldn't have to be defined multiple times in the HTML?

Reply:
I've changed this now, by registering a new general purpose template tag settings_value that can provide any settings value to templates (without having to add that settings value to every view that needs it). I'll add it to the django template project too, as will be useful for all projects to have it.